projects/samples/contests/robocup/README.md

```diff
@@ -9,22 +9,40 @@ You will also need to get familiar with Webots by reading the [Webots User Guide
 
 ## Installation
 
-1. [Build Webots from the source](https://github.com/cyberbotics/webots/wiki) from the [feature-robocup-controllers](https://github.com/cyberbotics/webots/tree/feature-robocup-controllers) branch (recommended) or install the latest nightly build of [Webots R2021b](https://github.com/cyberbotics/webots/releases) and checkout the content of this folder locally.
-2. Build the latest version of the [GameController](https://github.com/sheepsy90/GameController).
-3. Install the construct Python module: `pip install construct`.
-4. Define the `GAME_CONTROLLER_HOME` environment variable to point to the `GameController` folder: `export GAME_CONTROLLER_HOME=/my/folder/GameController`
+1. [Build Webots from the source](https://github.com/cyberbotics/webots/wiki) from the [release](https://github.com/RoboCup-Humanoid-TC/webots/tree/release) branch on the official RoboCup Humanoid TC fork: https://github.com/RoboCup-Humanoid-TC/webots
+2. Build the latest version of the official RoboCup Humanoid TC fork of the [GameController](https://github.com/RoboCup-Humanoid-TC/GameController).
+   ```
+   git clone https://github.com/RoboCup-Humanoid-TC/GameController
+   cd GameController
+   ant
+   ```
+3. Install Python dependencies:
+   ```
+   cd webots/projects/samples/contests/robocup/controllers/referee
+   pip install -r requirements.txt
+   ```
+4. Build the controllers:
+   ```
+   apt-get install protobuf-compiler libprotobuf-dev
+   cd webots/projects/samples/contests/robocup
+   WEBOTS_HOME=/path/to/webots make
```

Comment:
This variable should be set when webots is installed no?

Reply:
I purposely passed this var here. I think it is very easy to forget to set this env var (I forgot to set it many times). I think the most fail-safe way here is to provide it. It will not hurt anybody to pass it here again but will help people who otherwise wouldn't understand why `make` fails with a weird error message.

Reply:
Hm okay, I adjusted it.

However, to make it very obvious on how to install Webots (including how to set WEBOTS_HOME), I also put the build instructions I always use in the README now. Especially updating Webots from source can be tricky, if you don't know how to do it.